src/video_core/query_cache/query_stream.h

```diff
@@ -0,0 +1,139 @@
+// SPDX-FileCopyrightText: Copyright 2023 yuzu Emulator Project
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#pragma once
+
+#include <deque>
+#include <optional>
+#include <vector>
+
+#include "common/assert.h"
+#include "common/common_types.h"
+#include "video_core/query_cache/bank_base.h"
+#include "video_core/query_cache/query_base.h"
+
+namespace VideoCommon {
+
+class StreamerInterface {
+public:
+    StreamerInterface(size_t id_) : id{id_}, dependence_mask{}, dependent_mask{} {}
```

Comment:
```suggestion
    explicit StreamerInterface(size_t id_) : id{id_} {}
```

The empty initializers can be replaced by just initializing the members directly